app/assets/javascripts/discourse/app/components/sidebar/section.js

```diff
@@ -8,7 +8,8 @@ export default class SidebarSection extends Component {
   @service keyValueStore;
 
   @tracked displaySectionContent;
-  collapsedSidebarSectionKey = `sidebar-section-${this.args.sectionName}-collapsed`;
+  sidebarSectionID = `sidebar-section-${this.args.sectionName}`;
```

Comment:
`sidebar-section-${this.args.sectionName}` is already being used as the class name for the sidebar section.